### PR TITLE
ci: Stop setting CLOUDSDK_PYTHON for system_tests/noxfile.py

### PR DESCRIPTION
**Pull request by jameslynnwu**
google-cloud-sdk.tar.gz is now compatible with Python 3.

### CODE REVIEW

File: system_tests/noxfile.py [hunk 1]
```diff
@@ -70,11 +70,6 @@
 # The full path to the gcloud cli executable.
 GCLOUD = str(CLOUD_SDK_INSTALL_DIR.join("bin", "gcloud"))
 
-# gcloud requires Python 2 and doesn't work on 3, so we need to tell it
-# where to find 2 when we're running in a 3 environment.
-CLOUD_SDK_PYTHON_ENV = "CLOUDSDK_PYTHON"
-CLOUD_SDK_PYTHON = which("python2", None)
```
Comment (sai-sunder-s) [inline system_tests/noxfile.py:76]: Has gcloud stopped supporting 2.7 or is this change because gcloud supports python 3 as well, so we can just use it?

**Review by sai-sunder-s — APPROVED** [review]

**Review by parthea — APPROVED** [review]

File: system_tests/noxfile.py (hunk 1 above)
Comment (jameslynnwu) [reply system_tests/noxfile.py:76]: gcloud supports python 3 now, and we will stop supporting python 2 soon, so we shouldn't choose py2.

Comment (dandhlee) [reply system_tests/noxfile.py:76]: If it's "soon", shouldn't we still support until it is for sure deprecated?

**Comment by arithmetic1728** [comment]
close the PR as this is removed in #1278

**Pull request closed without merging**